packages/jest-haste-map/src/lib/docblock.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const commentEndRe = /\*\/$/;
+const commentStartRe = /^\/\*\*/;
+const docblockRe = /^\s*(\/\*\*(.|\r?\n)*?\*\/)/;
```

Comment:
fbsource contains modules that look like this:
```js
/*
 * @providesModule x
 */
```
(no double star)

and also contain `'use strict'` and single line comments before the docblock (although you have that covered)